main.go

```diff
@@ -42,26 +47,30 @@ var (
 	namespaceEnvVar = "WATCH_NAMESPACE"
 )
 
+type k8sManager interface {
```

Comment:
🚫 **[checkstyle]** <deadcode> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`k8sManager` is unused